src/main/java/com/openmodloader/api/IOmlContext.java

```diff
@@ -0,0 +1,16 @@
+package com.openmodloader.api;
+
+import net.fabricmc.api.Side;
+import net.minecraft.server.MinecraftServer;
+
+import javax.annotation.Nullable;
+import java.io.File;
+
+public interface IOmlContext {
```

Comment:
I'd call these GlobalContext's

Reply:
Or maybe `GameContext`?

Reply:
Could work yeah, I just want to avoid using "Oml" in class names

Reply:
Fair enough. I think there are a few other cases like `OMLMixinProvider`. Should we change these?